apps/vx-scan/backend/package.json

```diff
@@ -94,24 +85,19 @@
     "@types/zip-stream": "workspace:*",
     "@typescript-eslint/eslint-plugin": "^5.37.0",
     "@typescript-eslint/parser": "^5.37.0",
-    "esbuild": "^0.14.29",
-    "esbuild-runner": "^2.2.1",
     "eslint": "^8.23.1",
     "eslint-config-prettier": "^8.5.0",
     "eslint-import-resolver-node": "^0.3.4",
-    "eslint-plugin-cypress": "^2.12.1",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-jest": "^26.1.5",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-vx": "workspace:*",
-    "fast-check": "^2.23.2",
     "is-ci-cli": "^2.1.2",
     "jest": "^26.6.3",
     "jest-junit": "^14.0.1",
     "jest-watch-typeahead": "^0.6.4",
     "lint-staged": "^10.5.3",
     "nock": "^13.1.0",
-    "nodemon": "^2.0.20",
```

Comment:
This was removed too aggressively and `pnpm start` in the `frontend` no longer works.

Reply:
I am running in to this issue on my branch.

Reply:
Sorry about that! Fixed in #2824